cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -159,6 +153,18 @@ func (o *ReleaseOptions) releaseResources(appSpec kcv1alpha1.AppSpec, pkgBuild c
 	return nil
 }
 
+func generateValuesSchema(pkgBuild cmdpkgbuild.PackageBuild) (*kcdatav1alpha1.ValuesSchema, error) {
+	var yttPaths []string
+	for _, templateStage := range pkgBuild.Spec.Template.Spec.App.Spec.Template {
+		if templateStage.HelmTemplate != nil {
+			return NewHelmValuesSchemaGen(templateStage.HelmTemplate.Path).Schema()
```

Comment:
What should happen if there are more templating step after helm?

Reply:
Any number of combinations are possible. But since we are saying that tool will handle only one fetch config, I dont think we can support complex use case.

Reply:
What are your thoughts?